app/renderer/components/urlBarIcon.js

```diff
@@ -11,6 +11,19 @@ const dndData = require('../../../js/dndData')
 const {isSourceAboutUrl} = require('../../../js/lib/appUrlUtil')
 const searchIconSize = 16
 
+const getIconCssClass = (ctx) => {
+  if (ctx.isSearch) {
```

Comment:
k. But i dont think you need to make this change since now its if-else so only one class gets returned by this function. So you can never have a `fa-search fa-list` together.